internal/incident/incident.go

```diff
@@ -147,6 +151,11 @@ func (i *Incident) ProcessEvent(ctx context.Context, ev *event.Event) error {
 		return errors.New("can't insert incident event to the database")
 	}
 
+	if err := i.handleMuteUnMute(ctx, tx, ev); err != nil {
+		i.logger.Errorw("Cannot insert incident muted history", zap.String("event", ev.String()), zap.Error(err))
+		return errors.New("cannot insert incident muted history")
```

Comment:
Why not wrap the original error?

Reply:
The original error is logged with `zap.Error()` and we shouldn't be returning database-related errors to the clients, but #188 may make this more consistent.